.circleci/config.yml

```diff
@@ -35,7 +35,32 @@ jobs:
     - run:
         name: Perform build
         command: |
-          go build -o server *.go
+          go build -o generator *.go
+
+  # This job needs to be in machine because it has COPY inside the Dockerfile
+  # which cannot be handled in the docker type jobs
+  createImage:
+    machine: true
+
+    steps:
+      - checkout
+
+      - run:
+          name: Build Docker image
+          command: |
+            tagRE='release-(.*)'
+            if [[ "${CIRCLE_TAG}" =~ ${tagRE} ]]; then
+              VERSION=${BASH_REMATCH[1]}
+
+              docker build -t spring-boot-generator:${VERSION} -f docker/Dockerfile_generator .
```

Comment:
@cmoulliard Good point. Actually in the previous iterations of my implementation, I didn't use it. But I still did have to recompile the project (in addition to what release does), because the generic amd64 binary produced by the release process does not work on the busybox image.

So I believe that this is the simplest and most clear solution at the moment

Reply:
So, let's keep it then ;-)